content/docs/studio/get-started.md

```diff
@@ -6,9 +6,15 @@ Studio.
 https://www.youtube.com/watch?v=hKf4twg832g
 
 1. Sign in to your [Iterative Studio](https://studio.iterative.ai/) dashboard
-   using GitHub, GitLab or Bitbucket.
+   using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
```

Comment:
Not sure we needed the .com/org s but OK 🙂

Reply:
I added these to distinguish them from on-prem/custom GitLab (and maybe GitHub in the future).

Reply:
I see.